src/renderer/views/Settings/Settings.vue

```diff
@@ -27,6 +27,8 @@
       <hr v-if="usingElectron">
       <experimental-settings v-if="usingElectron" />
       <hr>
+      <ryd-settings />
```

Comment:
It might make more sense placing it next to the sponsor block settings as they are both settings for external services.
```suggestion
      <hr>
      <ryd-settings />
      <hr v-if="usingElectron">
      <experimental-settings v-if="usingElectron" />
```